datasets/librispeech/_metadata.json

```diff
@@ -0,0 +1,254 @@
+{
+    "@context": {
+        "@vocab": "https://schema.org/",
+        "ml": "http://mlcommons.org/schema/"
+    },
+    "@type": "Dataset",
+    "@language": "en",
+    "name": "LibriSpeech ASR corpus",
+    "url": "http://www.openslr.org/12",
+    "description": "LibriSpeech is a corpus of approximately 1000 hours of 16kHz read English speech, prepared by Vassil Panayotov with the assistance of Daniel Povey. The data is derived from read audiobooks from the LibriVox project, and has been carefully segmented and aligned.\nAcoustic models, trained on this data set, are available at kaldi-asr.org and language models, suitable for evaluation can be found at http://www.openslr.org/11/.\nFor more information, see the paper \"LibriSpeech: an ASR corpus based on public domain audio books\", Vassil Panayotov, Guoguo Chen, Daniel Povey and Sanjeev Khudanpur, ICASSP 2015 (submitted) (pdf)",
```

Comment:
Alternatively you could add a "citation"  instead of the "see the paper". I would have a slight preference for that.

The current solution is fine by me as well. Now the description is directly copied from the original site, which is probably what most people will do in practice anyway.